Find a functionally equivalent code:android.content.res.BridgeTypedArray.getString:COMMENT
Method Modifier: public      
Comment:/**
 * Retrieve the string value for the attribute at <var>index</var>.
 *
 * @param index Index of attribute to retrieve.
 *
 * @return String holding string data.  Any styling information is
 * removed.  Returns null if the attribute is not defined.
 */

@@ -1,9 +1,10 @@
 {
-    if (index < 0 || index >= mResourceData.length) {
+    if (!hasValue(index)) {
         return null;
     }
-    if (mResourceData[index] != null) {
-        return mResourceData[index].getValue();
-    }
-    return null;
+    // As unfortunate as it is, it's possible to use enums with all attribute formats,
+    // not just integers/enums. So, we need to search the enums always. In case
+    // enums are used, the returned value is an integer.
+    Integer v = resolveEnumAttribute(index);
+    return v == null ? mResourceData[index].getValue() : String.valueOf((int) v);
 }

